src/Analyzer.Core.UnitTests/TemplateAnalyzerTests.cs

```diff
@@ -54,6 +58,51 @@ public void AnalyzeTemplate_ValidInputValues_ReturnCorrectEvaluations(string res
             Assert.AreEqual(expectedEvaluationPassCount, evaluationsWithResults.Count(e => e.Passed));
         }
 
+        [DataTestMethod]
+        [DataRow("SimpleNestedFail.json", new int[] { 36, 43, 46, 52, 53, 54 }, DisplayName = "Simple nested template example")]
+        [DataRow("DoubleNestedFail.json", new int[] { 31, 37, 53, 59, 60,  61}, DisplayName = "Nested templates with two levels")]
+        [DataRow("InnerOuterScopeFail.json", new int[] { 49, 55, 56, 101, 107, 108, 109 }, DisplayName = "Nested template with inner and outer scope, with colliding parameter names in parent and child templates")]
+        [DataRow("ParameterPassingFail.json", new int[] { 53, 59, 62, 68, 69 }, DisplayName = "Nested template with parameters passed from parent")]
+        public void AnalyzeTemplate_ValidNestedTemplate_ReturnsExpectedEvaluations(string templateFileName, dynamic lineNumbers)
+        {
+            string filePath = Path.Combine(".", "templates", templateFileName);
```

Comment:
I think the "." is not needed, can you double check?

Reply:
I did. It is not needed indeed. I got rid of it